typescript/sdk/src/ism/HyperlaneIsmFactoryDeployer.ts

```diff
@@ -33,21 +33,33 @@ export class HyperlaneIsmFactoryDeployer extends HyperlaneDeployer<
   async deployContracts(
     chain: ChainName,
   ): Promise<HyperlaneContracts<IsmFactoryFactories>> {
-    const multisigIsmFactory = await this.deployContract(
+    const merkleRootMultisigIsm = await this.deployContract(
```

Comment:
I'm confused by the name changes here. We're deploying factories, no?

Reply:
urmmmm